tasks/prime.py

```diff
@@ -615,24 +615,87 @@ def update_mto_shipment_status(self, overrides=None):
         if not mto_shipment:
             logger.debug("updateMTOShipmentStatus: ⚠️ No mto_shipment found.")
             return None  # can't run this task
+        # To avoid issues with the mto shipment being stale
+        # retrieve the move associated with the shipment
+        # and then use the newly fetched move to the find most up to date version of the shipment
+        move_id = mto_shipment["moveTaskOrderID"]
+        headers = {"content-type": "application/json"}
+        resp = self.client.get(
```

Comment:
Do you know the root cause of the failure? Is there another endpoint that isn't updating the local store properly?  Do we know it's not just a race condition depending on the number of users and moves?

Reply:
I'm not 100% sure, I'm leaning more towards a race condition because when I checked if any test was wasn't updating the local store properly nothing surfaced. Even when I had most test commented out and it was still an issue

Reply:
Yea in a previous PR I was commenting out other tests too and only running ~2 users to figure out if it was a bug or just random contention.  It turned out that the move eTag being returned was a bug in mymove in that case.  This will count in the stats to the support API GET move-task-order requests too.